README.md

```diff
@@ -49,7 +49,7 @@ The balenaDash project includes [wifi-connect](https://github.com/balena-io/wifi
 
 WiFi Connect periodically tests for a functional internet connection, if nothing is found the device sets itself up as a WiFi access point named `balenaDash` that you can join with a mobile device. **Note: due to the fact the built-in WebUI for balenaDash uses port 80, the default captive portal popup screen will not work.**
```

Comment:
The bit about the captive portal not working is redundant now, right?

Reply:
It is!

Reply:
Removing!